docs/UserGuide.md

```diff
@@ -339,16 +303,18 @@ Format: `exit`
 
 ### Saving the data
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+iVolunteer data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
```

Comment:
i thought data is saved after any command? i read from LogicManager::execute

Reply:
I think we should save data only after commands that change the data. Easier to implement the undo feature (which i wanna do down the line) :)